gluster-colonizer.py

```diff
@@ -1122,7 +1122,7 @@ def autoNodeInformation():
             while True:
                 try:
                     host_command('/bin/systemctl start NetworkManager')
-                    p1 = Popen(shlex.split('/bin/nmcli con show %s' % nm_mgmt_interface), stdout=PIPE)
+                    p1 = Popen(shlex.split('/bin/nmcli d show %s' % nm_mgmt_interface), stdout=PIPE)
```

Comment:
I'm not sure about this. Right now, for bonds and teams created under "standard" conditions, we expect a bond- or a team- prefix for the device name for nmcli commands against the connection. And AFAICT doing a 'nmcli dev show' doesn't output the IP4.ADDRESS that we are grepping for in the line below.